scripts/deploy-multus-network.sh

```diff
@@ -28,6 +28,10 @@ if $TNF_NON_OCP_CLUSTER; then
 	./generate_yamls.sh
```

Comment:
I also added a wrapper for OS type around the call to `./get_tools.sh` because it only downloads linux binaries.

Reply:
Something like:
```
if [ "${ARCH}" != "arm64" ]; then
    ./get_tools.sh
fi
```

Reply:
Fixed